GitUI/Editor/FileViewer.cs

```diff
@@ -530,19 +530,19 @@ public void ViewCurrentChanges(GitItemStatus item, bool isStaged, [CanBeNull] Ac
             ThreadHelper.JoinableTaskFactory.RunAsync(
                 async () =>
                 {
-                    if (!item.IsTracked)
+                    if (!item.IsSubmodule)
                     {
-                        await ViewFileAsync(item.Name, item.IsSubmodule, openWithDifftool);
-                        SetVisibilityDiffContextMenuStaging();
```

Comment:
Can the call of `SetVisibilityDiffContextMenuStaging` really be removed?
If yes, the method itself should be removed. Or will it be reused in a follow-up PR?

Slightly off-topic: FileStatusList context menu:
I tried to `cherry pick changes` from a new file in an unmerged branch (like "bugfix/i6723-filename-space-2") without effect.
I guess this menu item is not applicable for new files. The same applies to `Reset file(s) to | Selected`.

Reply:
> Can the call of SetVisibilityDiffContextMenuStaging really be removed?
If yes, the method itself should be removed. Or will it be reused in a follow-up PR?

SetVisibilityDiffContextMenuStaging is not removed, just moved around.
However #7825 removes SetVisibilityDiffContextMenuStaging  and a follow up to that will remove ViewCurrentChanges() and let all "patchviewers" use ViewChangesAsync()
The unification is why I prefer to look over this after #7825

Reply:
Sounds good.